src/clients/__test__/stormGlass.test.ts

```diff
@@ -46,7 +48,9 @@ describe('StormGlass client', () => {
       data: incompleteResponse,
     } as HTTPUtil.Response);
 
-    const stormGlass = new StormGlass(mockedRequest);
+    mockedCache.get = jest.fn().mockReturnValue(undefined);
```

Comment:
I assume you are doing this because the previous test added the response to the cache right? It shouldn't happen, one test shouldn't impact the other, I think we should pass the mockedCache in the previous test as well just to make sure that the cache won't get created.

Reply:
Nice, I'll change it!